packages/app/config/environments.json

```diff
@@ -23,6 +23,14 @@
         "uri": "http://localhost:8080/graphql/api/v1",
         "type": "elide",
         "suggestedDataTables": []
+      },
```

Comment:
Can we model namespaces like this:

```json
      {
        "name": "corpAnalytics",
        "displayName": "Corp Analytics - General",
        "uri": "http://localhost:8080/graphql/api/v1",
        "type": "elide",
        "suggestedDataTables": [] // for default namespace
        "namespaces": [{
          "name": "rev",
          "displayName": "Revenue",
          "suggestedDataTables": ['allRevenue']
        }]        
      },
```

Then the datasource id for namespaces can be something like:
- `corpAnalytics.default` 
- `corpAnalytics.rev`